Packs/MicrosoftNPS/README.md

```diff
@@ -0,0 +1,36 @@
+This pack includes XSIAM content.
+
+## Collect Events from Vendor
+
+In order to use the collector, you can use one of the following options to collect events from the vendor:
+ - [Broker VM](#broker-vm)
+ - [XDRC (XDR Collector)](#xdrc-xdr-collector) 
+
+In either option, you will need to configure the vendor and product for this specific collector.
+
+### Broker VM
```

Comment:
Broker VM is not relevant for this collector as the collection method for NPS is XDRC, please remove this section.